cassandra/metadata.py

```diff
@@ -475,14 +475,16 @@ def make_token_replica_map(self, token_to_host_owner, ring):
         ring_len_range = range(ring_len)
```

Comment:
What the thinking behind having the variables:  `ring_len` and `ring_len_range`? They seem to be used once. Could the line there used on be simplified to?

```python
for i in range(len(ring)):
    pass
```